js/lightbeam.js

```diff
@@ -1,9 +1,53 @@
 async function renderGraph() {
-  const canvas = document.getElementById('canvas');
-  const context = canvas.getContext('2d');
   const websites = await store.getAll();
 
-  viz.draw(context, websites);
+  // reformat data for D3
```

Comment:
This looks like it is coupled to the visualisations. It shouldn't be.

This code might:
- Change hostname paths and clean replicated data
- Query first party and third parties in a an order it chooses

It shouldn't be changing output to match how a rendering might display it.